api/src/main/java/org/apache/unomi/api/Event.java

```diff
@@ -369,10 +372,12 @@ public void setProperties(Map<String, Object> properties) {
     /**
      * @return the scope
      */
+    @Deprecated
```

Comment:
why **@Deprecated** the scope of the event?

Reply:
According to the [ticket](https://issues.apache.org/jira/browse/UNOMI-350)

Reply:
Great, thank you